### DISCUSSION TITLE
大括号怎么上移指针？How do I move the pointer up with braces?以上翻译结果来自有道神经网络翻译（YNMT）· 通用场景

### DISCUSSION THREAD

**Discussion by WanderingSta** [jgraph/drawio#4305] [Q&A]
![1](https://github.com/jgraph/drawio/assets/112313941/913b58ea-c70c-4e7a-9af1-d72922d9f2b5)

**Comment by AuKeeWa** [comment]
您好，请问您现在清楚大括号该如何上移指针了吗？我也有同样的疑问，求解答，谢谢！
